sdk/tables/data-tables/src/utils/batchHeaders.browser.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { getBaseBatchHeaders } from "./baseBatchHeaders";
+
+/**
+ * @internal
+ * Builds an object with the required headers for a Batch request. For the Browser
+ */
+export const getBatchHeaders = getBaseBatchHeaders;
```

Comment:
I wonder if it would be easier to write a function that filters out "unsafe" headers and apply that here, rather than relying on folks knowing to only add them to the node version?